cvIgnacioCalahorro.html

```diff
@@ -123,7 +115,7 @@ <h2 class='about'>ABOUT ME:</h2>
                         &nbsp;&nbsp;&nbsp;&nbsp;Jaén University. Graduated 2015
                     </p>
                     <p>
-                        &nbsp;&nbsp;My final project was to develop the topographic base map for a 100 ha of new developable land, to integrate it into the road network from the town planning authority, and to define the alignment and profile in order to calculate the volume of earthworks, the budget and the altered or modified topography 
+                        &nbsp;&nbsp;My final project was to develop the topographic base map for 100 ha of new developable land, to integrate it into the road network from the town planning authority, and to define the alignment and profile in order to calculate the volume of earthworks, the budget and the altered or modified topography. 
```

Comment:
- The "a" before "100 ha" shouldn't be there
- Missing dot at the end of the sentence
- Otherwise pure beauty